### PR TITLE
Add unit test for common util

### LINKED ISSUE #48: Add unit test for common util
## Issue Description

Type: *feature request*
Add unit test for common util

### Describe what feature you want
Code path: util/*
Target coverage: 50+% statements.

### PR DESCRIPTION
**Pull request by zsy619**
### Describe what this PR does / why we need it
Add unit test for util package
 
### Does this pull request fix one issue?

Fixes #48 

### Special notes for reviews

### CODE REVIEW

**Comment by louyuting** [comment]
Hi, it indicate the CI build failed.
Could you please fix it.
Thanks~

File: util/time_test.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
+package util
+
+import (
+	"fmt"
+	"testing"
+)
+
+//要考虑时区
```
Comment (sczyh30) [inline util/time_test.go]: It's better to write English comments here :)

Comment (zsy619) [reply util/time_test.go]: ok

Comment (zsy619) [reply util/time_test.go]: already fix

File: util/time_test.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			"test1",
+			args{1582421778887},
+			"2020-02-23 01:36:18", //"2020-02-23 09:36:18",
```
Comment (sczyh30) [inline util/time_test.go]: These test cases will fail in non-UTC-zone machines. We may need to subtract the time zone offset. For example:

```go
_, offset := time.Now().Zone()
offsetMs := uint64(offset * 1000)
```

Comment (zsy619) [reply util/time_test.go]: 👌

Comment (zsy619) [reply util/time_test.go]: already fix

**Comment by sczyh30** [comment]
Thanks for contributing! I'll improve some of the code comments later.

**Pull request merged**